haiku-games/lightsoff/lightsoff-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="A puzzle game based on the original handheld game Lights Out"
+DESCRIPTION="LightsOff is a redition of the addictive puzzle game Lights Out."
+APP="LightsOff"
+#HOMEPAGE="https://github.com/HaikuArchives/$APP"
+HOMEPAGE="https://github.com/owenca/$APP"
+COPYRIGHT="2005 DarkWyrm"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="e607ee6fca45c620b5f54b247254edbcd71cd33d"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="c1df23b0100ad86b684c078de5132e83a216ed317d61670c969ab67e7d8a9022"
+SOURCE_FILENAME="$APP-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="$APP-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
+
+PROVIDES="
+	lightsoff = $portVersion
+	app:$APP = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
```

Comment:
```diff
 	"
 
+defineDebugInfoPackage lightsoff \
+	$appsDir/$APP
+
 BUILD()
```

Can you please add a debuginfo package?
Thanks!

Reply:
Done. :)